runtime-common/src/messages.rs

```diff
@@ -1031,19 +1029,33 @@ mod tests {
 			unreachable!()
 		}
 	}
-
 	impl BridgedChainWithMessages for BridgedChain {
 		fn maximal_extrinsic_size() -> u32 {
 			BRIDGED_CHAIN_MAX_EXTRINSIC_SIZE
 		}
 
-		fn message_weight_limits(message_payload: &[u8]) -> RangeInclusive<Self::Weight> {
+		fn verify_dispatch_weight(message_payload: &[u8], payload_weight: &Weight) -> bool {
 			let begin =
 				std::cmp::min(BRIDGED_CHAIN_MAX_EXTRINSIC_WEIGHT, message_payload.len() as Weight);
-			begin..=BRIDGED_CHAIN_MAX_EXTRINSIC_WEIGHT
+			(begin..=BRIDGED_CHAIN_MAX_EXTRINSIC_WEIGHT).contains(payload_weight)
 		}
 	}
 
+	// fn test_lane_outbound_data() -> OutboundLaneData {
```

Comment:
Should we fix these tests?

Reply:
Leave it for now. It needs to mock a runtime with `pallet-fee-market`.